crates/clvm-derive/src/hide_values.rs

```diff
@@ -0,0 +1,45 @@
+use proc_macro2::TokenStream;
+use quote::ToTokens;
+use syn::{punctuated::Punctuated, Data, DeriveInput, Fields};
+
+use crate::parser::parse_clvm_options;
+
+pub fn impl_hide_values(mut ast: DeriveInput) -> TokenStream {
```

Comment:
I can't find the place where this is called from. I still don't understand why you need to mutate the AST here. we just use the AST do implement the `ClvmEncode` and `ClvmDecode` traits anway. Couldn't you just ignore the hidden fields in those functions? As if they had been removed from the AST